LanguageFeatures/Class-modifiers/syntax_mixin_A06_t01.dart

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion Mixin can be implemented and mixed in but not constructed or
+/// extended and is not exhaustive
+///
+/// @description Checks that a `mixin` is not exhaustive
+/// @author sgrekhov22@gmail.com
+
+// SharedOptions=--enable-experiment=class-modifiers
+
+import "class_modifiers_lib.dart";
+
+mixin _M {}
+class ImplementsM implements _M {}
+class WithM with _M {}
+
+String test1(_M c) => switch (c) {
+//                    ^^^^^^
+// [analyzer] unspecified
+// [cfe] unspecified
+      ImplementsM _ => "ImplementsM",
+      WithM _ => "WithM"
```

Comment:
Again just wondering why we don't get the UNREACHABLE message here.

Reply:
(Closing this thread, the point was that the UNREACHABLE cases were `sealed` classes.)